config/cloudkeeper.yml

```diff
@@ -24,4 +24,5 @@ cloudkeeper:
   logging:
     level: ERROR # Logging level
     file: /var/log/cloudkeeper/cloudkeeper.log # File to write log to. To turn off file logging leave this field empty.
+  lock-file: /var/lock/cloudkeeper # File used to ensure only one running instance of cloudkeeper
```

Comment:
Since I can change this during installation, it's not a problem ... but as a default it doesn't make sense. Only `root` would be able to create a file directly in `/var/lock/`. It would make more sense to have `/var/lock/cloudkeeper/cloudkeeper.lock`. Then I could create the parent directory with sufficient permissions for the `cloudkeeper` user to be able to write there.